helm/README.md

```diff
@@ -13,8 +13,7 @@
     ./deploy_airbyte_module_client_pod.sh
     ```
 
-    This command deploys a client pod. You get a `bash` prompt, from which you can run:
+    Now that the client pod is deployed, you can obtain the `letter_frequency` dataset from the airbyte-module server by running:
```

Comment:
Perhaps add an explanation of the test being performed: dataset used, what the specific airbyte module does, and the expected output.